melodies_monet/driver.py

```diff
@@ -600,14 +601,12 @@ def open_model_files(self, time_interval=None, control_dict=None):
                 self.obj = mio.models.raqms.open_mfdataset(file_list,**self.mod_kwargs)
             else:
                 self.obj = mio.models.raqms.open_dataset(file_list)
+            self.obj = self.obj.rename({'ptrop':'pres_pa_trop'})
```

Comment:
Should we do this in the reader instead?

For compat, you could change it to this:
```suggestion
            if 'ptrop' in self.obj and 'pres_pa_trop' not in self.obj:
                self.obj = self.obj.rename({'ptrop':'pres_pa_trop'})
```

Reply:
Needed to apply your change while I was testing the small OMPS pairing updates due to the addition of the variable limiting for RAQMS. 

Would it be better to recommend doing this renaming through the yaml file for model output containing the tropopause pressure?

Reply:
Seems to me that it would fit well [here in the reader](https://github.com/noaa-oar-arl/monetio/blob/05b5dba9038453ec289d33cbe50ba765f85b7d9d/monetio/models/raqms.py#L164) ? So it's done automatically if ptrop is there but no error otherwise.